<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>120</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Color</string>
    </property>
    <widget class="QRadioButton" name="radioButton_black">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Black</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_red">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Red</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_blue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Blue</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Effect</string>
    </property>
    <widget class="QCheckBox" name="checkBox_italic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Italic</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_bold">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bold</string>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_slider_value">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_replace_src">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_replace_dst">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_replace">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>replace</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>457</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
